packages/replay-next/components/inspector/values/ObjectRenderer.tsx

```diff
@@ -15,22 +13,11 @@ const MAX_PROPERTIES_TO_PREVIEW = 5;
 export default function ObjectRenderer({ context, object, pauseId }: ObjectPreviewRendererProps) {
   const { className, preview } = object;
 
-  const [properties, propertiesWereTruncated] = useMemo(() => {
-    if (preview == null) {
-      return [[], false];
-    }
-
-    return mergePropertiesAndGetterValues(
-      preview.properties || [],
-      preview.getterValues || [],
-      MAX_PROPERTIES_TO_PREVIEW
-    );
-  }, [preview]);
-
-  const showOverflowMarker = object.preview?.overflow || propertiesWereTruncated;
-
+  const properties = preview?.properties?.slice(0, MAX_PROPERTIES_TO_PREVIEW);
```

Comment:
We were previously showing a mix of properties and getter values for collapsed objects.

This change means that objects like this...
![Screenshot 2024-01-30 at 12 46 31 PM](https://github.com/replayio/devtools/assets/29597/76f62e5b-12da-487a-932d-0aa16aae7a6f)

Might now be rendered like this...
![Screenshot 2024-01-30 at 12 46 34 PM](https://github.com/replayio/devtools/assets/29597/59314823-cf41-4247-9a3e-6ffbf3819733)

This seems like a regression. I think it's also why the [`console_async` test](https://tests.replay.io/recording/consoleasync-support-console-evaluations-in-async-frames--aef9d33f-2311-4218-8a2e-3db94a3deecd?e2e=&apiKey=&commentId=&focusWindow=eyJiZWdpbiI6eyJwb2ludCI6IjI1OTYxNDg0Mjk1MTQ4NTcyODYzNDg0ODM0ODMxNDAzMDciLCJ0aW1lIjoxMjUxfSwiZW5kIjp7InBvaW50IjoiOTM0NjEzNDM0NjQyNzY1MzQ3MDg2OTQ3ODQ4NDU4MDc2MTYiLCJ0aW1lIjo2MTI5M319&point=89891639374026608437481669868407782&primaryPanel=cypress&secondaryPanel=console&time=58427&viewMode=non-dev) failed.

Reply:
Both Firefox and Chromium debuggers show only own properties in the inline preview, so I've copied that behavior.
The test is failing because the underlying example recording was made before RUN-3135 was fixed, so the own properties `message` and `stack` didn't show up in `properties` even though they should.

Reply:
> Both Firefox and Chromium debuggers show only own properties in the inline preview, so I've copied that behavior.

Got it 👍🏼 

> The test is failing because the underlying example recording was made before RUN-3135 was fixed, so the own properties message and stack didn't show up in properties even though they should.

Gotcha. I guess we're kind of stuck until that fix has been released then?

I guess we can disable the failing part of the test and file a blocked FE issue to update the recording once RUN-3135 has shipped.

Reply:
RUN-3135 has already been shipped in the `20240126` release but the example was still recorded with the `20240123` release. I've updated most examples in #10249.

Reply:
I didn't even know we had a `20240126` release 😫 I wish it was easier to keep up with release status